gazebo_ros/test/test_sim_time.cpp

```diff
@@ -51,35 +53,48 @@ TEST_F(TestSimTime, TestClock)
   using ClockMsg = rosgraph_msgs::msg::Clock;
 
   // Create a node which will use sim time
-  std::vector<std::string> args;
-  std::vector<rclcpp::Parameter> params = {rclcpp::Parameter("use_sim_time", true)};
-  rclcpp::NodeOptions node_options;
-  node_options.arguments(args);
-  node_options.initial_parameters(params);
-  auto node = std::make_shared<rclcpp::Node>("test_sim_time", "", node_options);
-
-  // Get two clock messages, caching the ROS time once each is received
-  auto first_msg =
-    gazebo_ros::get_message_or_timeout<ClockMsg>(node, "/clock", rclcpp::Duration(5, 0));
-  ASSERT_NE(first_msg, nullptr);
-  auto first_msg_time = node->now();
-  auto second_msg =
-    gazebo_ros::get_message_or_timeout<ClockMsg>(node, "/clock", rclcpp::Duration(1, 0));
-  ASSERT_NE(second_msg, nullptr);
-  auto second_msg_time = node->now();
-
-  // The SIM time should be close to zero (start of simulation)
-  EXPECT_LT(rclcpp::Time(first_msg->clock), rclcpp::Time(1, 0, RCL_ROS_TIME));
-  EXPECT_LT(rclcpp::Time(second_msg->clock), rclcpp::Time(1, 0, RCL_ROS_TIME));
-  // Time should go forward
-  EXPECT_GT(rclcpp::Time(second_msg->clock), rclcpp::Time(first_msg->clock)) <<
-    rclcpp::Time(second_msg->clock).nanoseconds() << "    " <<
-    rclcpp::Time(first_msg->clock).nanoseconds();
-  // The message from the clock should be the node's time
-  EXPECT_EQ(first_msg_time, rclcpp::Time(first_msg->clock)) << first_msg_time.nanoseconds() <<
-    "    " << rclcpp::Time(first_msg->clock).nanoseconds();
-  EXPECT_EQ(second_msg_time, rclcpp::Time(second_msg->clock)) << second_msg_time.nanoseconds() <<
-    "    " << rclcpp::Time(second_msg->clock).nanoseconds();
+  auto node = std::make_shared<rclcpp::Node>("test_sim_time");
+  node->set_parameter(rclcpp::Parameter("use_sim_time", true));
```

Comment:
You may prefer it as-is because it's more explicit, but you can do it slightly shorter with an initializer list:

```c++
node->set_parameter({"use_sim_time", true});
```